app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :idea
+  validates :user_name, :body, presence: true
+  scope :ordered_by_newest, -> { order('created_at DESC NULLS LAST')}
```

Comment:
`NULLS LAST` is it necessary?

Reply:
Yep, because we want to have comments with 'created_at: NULL' appear at the bottom of comments view.